Dockerfile.ci

```diff
@@ -1,5 +1,5 @@
 # Step one: build compliance-operator
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-release-golang-1.17-openshift-4.10 AS  builder
```

Comment:
Are you sure this is the correct reference? Because CI fails with:
```
error: build error: failed to pull image: After retrying 2 times, Pull image still failed due to error: Error initializing source docker://registry.ci.openshift.org/ocp/builder:rhel-8-release-golang-1.17-openshift-4.10: Error reading manifest rhel-8-release-golang-1.17-openshift-4.10 in registry.ci.openshift.org/ocp/builder: manifest unknown: manifest unknown
```